### DISCUSSION TITLE
Let everyone can edit wiki?

### DISCUSSION THREAD

**Discussion by C-BJ** [inanevin/LinaEngine#71] [Ideas]
Now no one can edit wiki except @inanevin 
Wiki should be miscellaneous documents
Common documents should be placed in the repository or future websites

**Comment by inanevin** [comment]
No, Wiki should be the documentation of Lina Engine project. If a documentation exists on a separate website, wiki should be either empty or it should be high-level documentation such as FAQs, Links etc.
